### PR TITLE
Added homework B02 skeleton (no tests included)

### PR DESCRIPTION
**Pull request by friedalexuni**

### CODE REVIEW

File: src/adap-b01/names/Name.ts [hunk 1]
```diff
@@ -10,6 +10,7 @@ export class Name {
         throw new Error("needs implementation");
     }
 
+    // Returns human-readable representation of Name instance
```
Comment (joluj) [inline src/adap-b01/names/Name.ts:13]: It's highly recommended to use the [jsdoc](https://jsdoc.app/) standard for code documentation.
```suggestion
    /** Returns human-readable representation of Name instance */
```

File: src/adap-b02/names/Name.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+export const DEFAULT_DELIMITER: string = '.';
+export const ESCAPE_CHARACTER = '\\';
+
+export interface Name {
+
+    // Returns human-readable representation of Name instance
```
Comment (joluj) [inline src/adap-b02/names/Name.ts:6]: jsdoc
```suggestion
    /** Returns human-readable representation of Name instance */
```

**Review by joluj — APPROVED** [review]

**Comment by dirkriehle** [comment]
Auch fuer Einzeiler? Not really using JSDoc features here.

**Comment by joluj** [comment]
> Auch fuer Einzeiler? Not really using JSDoc features here.

IMO it's best to get used to language-specific conventions, even if we don't use JSDoc itself directly. For example, IDEs work better when following the JSDoc guidelines.

**Review by dirkriehle — COMMENTED** [review]
Thanks. So this is not yet a pull request, just an informal suggestion? Can I turn this into a pull request or do you have to do this?

**Comment by joluj** [comment]
It's not a pull request yet, it was a suggestion I made during the review for this PR (#7).
By adding the comments/suggestions and still approving the PR, we usually mean "This would be nice, but not required".

**Pull request merged**